main.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'app'
+
+def display_main_menu
+  puts "\nMain Menu: "
+  puts '1. List all books'
```

Comment:
- [ ] When the option to display all books is selcted you are required to display the title of the books and their authors, currently your app does not do this as shown in the image below :+1: 
- project requirement
 ![ab](https://github.com/AliRezaBmeDu/school-library/assets/89055502/a0f08613-3992-4f97-bb28-0c3b58121177)
- your project
![ab](https://github.com/AliRezaBmeDu/school-library/assets/89055502/9e6ac580-2654-4200-bdd1-f3aefc6a9aa7)

Reply:
done

Reply:
done